src/NuGet.Services.AzureSearch/Auxiliary2AzureSearch/UpdateDownloadsCommand.cs

```diff
@@ -171,6 +174,19 @@ private async Task<bool> PushIndexChangesAsync()
             return true;
         }
 
+        private async Task<SortedDictionary<string, SortedSet<string>>> GetPopularityTransfersAsync()
+        {
+            if (!_featureFlags.IsPopularityTransferEnabled())
```

Comment:
nit: personally I feel an enabled check is more readable and intuitive than not enabled..

Reply:
I prefer the negative pattern as it avoids nesting if there are multiple conditions that must be met before returning the result. For example:

```csharp
if (condition1)
{
    if (condition2)
    {
        return data;
    }
    else
    {
        return errorResult2;
    }
}
else
{
    return errorResult1;
}

return emptyResults;
```

Vs:

```csharp
if (!condition1)
{
    return errorResult1;
}

if (!condition2)
{
    return errorResult2;
}

return results;
```

Please let me know if you feel strongly about this.